yellowbrick/features/pcoords.py

```diff
@@ -169,6 +200,37 @@ def draw(self, X, y, **kwargs):
         Called from the fit method, this method creates the parallel
         coordinates canvas and draws each instance and vertical lines on it.
         """
+        # Convert from dataframe
+        if is_dataframe(X):
+            X = X.as_matrix()
+
+        # Choose a subset of samples
+        if isinstance(self.sample, int):
+            if self.sample < 1:
+                raise YellowbrickValueError(
+                    "`sample` parameter of type `int` must be greater than 1"
+                )
+            n_samples = min([self.sample, len(X)])
+        elif isinstance(self.sample, float):
+            if self.sample <= 0 or self.sample > 1:
+                raise YellowbrickValueError(
+                    "`sample` parameter of type `float` must be between 0 and 1"
+                )
+            n_samples = int(len(X) * self.sample)
+        else:
+            raise YellowbrickTypeError(
+                "`sample` parameter must be int or float"
+            )
+        X = X[:n_samples, :]
+
+        # Normalize
+        if self.normalize is not None:
+            if self.normalize not in self.normalizers:
+                raise YellowbrickValueError(
```

Comment:
Very, very good points. 

So I do consider these feature visualizers transformers; I want them to be useful in a pipeline to show exactly what's happening to the data.  The problem is, I think that if we do normalization in the visualizer but then don't pass that on, this could also be misleading. I guess this would mean that there is an argument for simply putting the normalizer transformer ahead of the pcoords in a pipeline rather than do something mysterious on the user's behalf.

I'll consider the idea of "visualizers are visualizers" and that they stand-alone without doing transformation, we've already walked down the road of "visual transformer == visualizer" but we're not so far down it we can't walk back. In the meantime, let's get these features merged!